src/fontra/client/core/glyph-controller.js

```diff
@@ -651,6 +655,33 @@ export class StaticGlyphController {
       }
     }
 
+    if (backgroundImageIndices.length) {
```

Comment:
While you are doing the overly generic `backgroundImageIndices`, might as well go all the way, and write it like this:
```js
    for (const i of backgroundImageIndices) {
      assert(i == 0, "we currently only support a single bg image");
      // Then, and check that now makes you go 
      //    return unionRect(...);  // lots of repetition
      // you can do "break;" instead
```
    }

Reply:
ok. I changed it, but named `i` to `imageIndex` to follow the same naming scheme similar to the code above: `anchorIndex` or `componentIndex`.